SCons/Node/Python.py

```diff
@@ -120,11 +127,8 @@ def get_contents(self) -> bytes:
         """Get contents for signature calculations."""
         return self.get_text_contents().encode()
 
-    def get_csig(self, calc=None):
-        """Because we're a Python value node and don't have a real
-        timestamp, we get to ignore the calculator and just use the
-        value contents.
-
```

Comment:
The closest thing to a "functional" change this PR was removing this optional argument. This is the only instance of `get_csig` taking an extra argument, and it's completely unused. Based of the docstring, I assume every `get_csig` took a `calc` argument at one point, with `Value` specifically treating it as a no-op; that might be why it was never properly cleaned up, remaining as a vestigal argument